pkgs/applications/editors/vim/plugins/nvim-treesitter/overrides.nix

```diff
@@ -51,6 +51,8 @@ in
   passthru = {
     inherit builtGrammars allGrammars grammarToPlugin withPlugins withAllGrammars;
 
+    vimPlugin = true;
```

Comment:
this is too ad-hoc. Doing   `passthru = (super.nvim-treesitter.passthru or { }) // { .. }` (like for meta) should be more-robust.